Index: tcl/board.tcl
===================================================================
--- tcl/board.tcl	(revision 3322)
+++ tcl/board.tcl	(working copy)
@@ -794,7 +794,7 @@
 
   array set ::toolbar_temp [array get ::toolbar]
 
-  set button_options {-height 20 -width 22 -command changeToolbar}
+  set button_options {-height 40 -width 44 -command changeToolbar}
 
   set pack_options {-side left -ipadx 1 -padx 3 -ipady 1}
 
@@ -1120,7 +1120,7 @@
 }
 
 # Double size the toolbar buttons (disabled)
-if {0} {
+if {1} {
   image create photo tempimage
   foreach i {ginfo glist pgn comment maint eco tree book crosst tmt engine
     hsearch bsearch msearch rfilter newgame copy paste
